docker-compose.yml

```diff
@@ -174,6 +255,20 @@ services:
     ports:
       - "19000:19000"
 
+  amc-frontend:
+    command: bash -c 'while true; do npm start; sleep 2; done'
+    container_name: tahoe.devstack.amc-frontend
+    # Allows attachment to the LMS service using 'docker attach <containerID>'.
+    stdin_open: true
+    tty: true
+    environment:
+      LMS_BASE: "localhost:18000"
```

Comment:
Do we still want to hardcode `localhost` in this case?

```
      LMS_BASE: "localhost:18000"
```

Reply:
This  is working as a charm, I don't see any reason to change it.